invenio_app/config.py

```diff
@@ -116,3 +116,6 @@ def defenders():
 APP_HEALTH_BLUEPRINT_ENABLED = True
 """Enable the ping (healthcheck) blueprint. (Default: ``False``)
 """
+
+APP_REQUEST_ID_HEADER_NAME = "X-Request-Id"
+"""Header name of nginx generated request id (uuid)"""
```

Comment:
``Header name containing a request id.`` 

The request id does not need to be generated by nginx, could be some other server. Also, it's not guaranteed to be a uuid.